.github/workflows/deploy-daily-reporter.yml

```diff
@@ -0,0 +1,26 @@
+name: Deploy Daily Reporter
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+permissions:
+  id-token: write
+
+jobs:
+  deploy_ingest_inspector:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: eu-west-1
+          role-to-assume: ${{ secrets.DEPLOY_INGEST_INSPECTOR_GHA_ROLE_ARN }}
```

Comment:
This role I think has sufficient permissions to perform this deployment and is arguably the correct role to use in any case given the relationship between the daily reporter and the ingest inspector.